gamestonk_terminal/portfolio_optimization/port_optimization_controller.py

```diff
@@ -46,7 +47,11 @@ def print_help(tickers: Set[str]):
         )
         print("")
         print("   add          add ticker to optimize")
-        print("   optimize     run optimization")
+        print("   select       overwrite current tickers with new tickers")
+        print("")
+        print("Optimization:")
```

Comment:
Is it possible to split these across their categories? Or you think there's no point?

Reply:
not sure I follow.  what categories?

Reply:
From what I read there are several common methods within "Mean Variance Optimization", but then there's also:
* Efficient Semivariance
* Efficient CVaR
* Black-Litterman Allocation
* Hierarchical Risk Parity
* Markowitz's critical line algorithm

Again, I haven't studied this in detail, so there may not be a "clean" cut where a nice structuring is possible. 

I'm thinking about taking the course that aia suggested to have a better idea of what we can/should do

Reply:
So I made it kinda like this.  Portflolio theory is intense and there are a lot of ways it can be done.  I think there are many ML approaches as well

I am not sure how intense we should be.  I haven't dug too much into the pyportfolioopt package, but it seems like there are ways you can add constraints (maybe max allocation or max factor exposure, etc), which seem like a good entry level for the average user.  

I was thinking of having the options for the user to do : max_sharpe , min_volatility and efficient frontier at a given risk level.  Then we can start trying to add on more complex methods (allowing for factors

Reply:
Yup, I agree with that approach.

It's hard for me to try to suggest a "cleanear approach" since I don't yet fully understand every moving piece.

But this sounds a great starting point to me